net_util/Cargo.toml

```diff
@@ -17,6 +17,7 @@ virtio-bindings = "0.1.0"
 vm-memory = { version = "0.6.0", features = ["backend-mmap", "backend-atomic", "backend-bitmap"] }
 vm-virtio = { path = "../vm-virtio" }
 vmm-sys-util = "0.8.0"
+getrandom = "0.2"
```

Comment:
Could you please sort it alphabetically?

Reply:
Fixed.